benchmarks/dusk_benchmarks.rs

```diff
@@ -213,13 +219,28 @@ mod edwards_benches {
                 Choice::from(1u8))))
         );
     }
+
+    pub fn bench_point_compression_decompression(c: &mut Criterion) {
+        c.bench(
+            "Compress/Decompress",
+            Benchmark::new("Point compression.",
+            |b| b.iter(|| P1_EXTENDED.compress()))
+        );
+
+        c.bench(
+            "Compress/Decompress",
+            Benchmark::new("Point decompression.",
+            |b| b.iter(|| P1_COMPRESSED.decompress().unwrap()))
+        );
+    }
 }
 
 criterion_group!(benchmarks, 
                 field_benches::bench_field_element_ops,
                 field_benches::bench_modular_inverse,
                 scalar_benches::bench_scalar_element_ops,
                 edwards_benches::bench_extended_point_ops,
-                edwards_benches::bench_projective_point_ops);
+                edwards_benches::bench_projective_point_ops,
+                edwards_benches::bench_point_compression_decompression);
 //criterion_group!(benchmarks, field_benches::bench_modular_inverse);
```

Comment:
Theres not a great deal for me to comment on here regarding the benchmarks but they are well documented. I think the use of lines from 239-244 allows users to see what they can see in advance so perhaps a restructuring, or having some better comments to the indication of the section would be useful.

Reply:
To indicate what? This is just the criterion group declaration, so I can collapse all benches on the Criterion main an run them with `cargo bench`.

Reply:
As discussed, I'll merge since this has not to be documented.